### DISCUSSION TITLE
FreeRTOS configMTIME_BASE_ADDRESS  is 0xFFFFFF90UL

### DISCUSSION THREAD

**Discussion by verigibest** [stnolting/neorv32#712] [Q&A]
Why configMTIME_BASE_ADDRESS of  neorv32/sw/example/demo_freeRTOS/FreeRTOSConfig.h is 0xFFFFFF90UL?

**Comment by stnolting** [comment]
We have reworked the entire address space in #648 - seems like we have forgotten to update the FreeRTOS files... I'll fix that ASAP :wink: Thanks for the hint!

**Reply by stnolting** [reply]
The MTIME base addresses are now fixed (https://github.com/stnolting/neorv32/commit/ef08d985fc01dbba125ddc8c9182de590fb27910) ✔️
